### ISSUE TITLE
Jetty claims to be running SNAPSHOT version

### ISSUE THREAD

**Issue by cowwoc** [eclipse/jetty.project#3918]
Version 9.4.15.v20190215

When hitting an HTTP 404 page I get:

`Powered by Jetty:// 9.4.z-SNAPSHOT` at the bottom of the page. Digging into the source-code, I see the following code in `Jetty.java`:

```
        if (pkg != null &&
                "Eclipse Jetty Project".equals(pkg.getImplementationVendor()) &&
                pkg.getImplementationVersion() != null)
            VERSION = pkg.getImplementationVersion();
        else
            VERSION = System.getProperty("jetty.version", "9.4.z-SNAPSHOT");
```

At runtime, I see that `pkg.getImplementationVersion()` returns null.

I suspect this is a bug.

My application runs on JDK 13 (build 31) and uses Java Modules. Is it possible that vendor information is being hidden due to the use of Java Modules?

**Comment by olamy** [comment]
I agree on this one.
What I would do is 
` 
VERSION = System.getProperty("jetty.version", __buildProperties.getProperty( "version", "9.4.z-SNAPSHOT"));
`
the file build.properties contains the version value coming from the pom so it sounds more accurate.
@gregw ?

**Comment by olamy** [comment]
fix in jetty-9.4.x and jetty-10.0.x branches.
